app/plugin_api.go

```diff
@@ -262,6 +262,10 @@ func (api *PluginAPI) GetChannelMember(channelId, userId string) (*model.Channel
 	return api.app.GetChannelMember(channelId, userId)
 }
 
+func (api *PluginAPI) GetChannelMembers(channelID string, page, perPage int) (*model.ChannelMembers, *model.AppError) {
```

Comment:
Can you lowercase `channelID` to `channelId` so it matches the other APIs?

Reply:
@jwilander Sure